hypershift-operator/metrics.go

```diff
@@ -50,6 +50,7 @@ type hypershiftMetrics struct {
 	nodePoolsWithFailureCondition      *prometheus.GaugeVec
 	hypershiftOperatorInfo             prometheus.GaugeFunc
 	hostedClusterTransitionSeconds     *prometheus.HistogramVec
+	hostedClusterCloudDanglingObjects  prometheus.GaugeFunc
```

Comment:
There's already a metric `SkippedCloudResourcesDeletionName = "hypershift_cluster_skipped_cloud_resources_deletion"` in the controller metrics, please use the same.

Also I don't really see where this is used?

Reply:
Sorry this is old code, I forgot to delete it.